src/nettests/ndt.js

```diff
@@ -169,8 +135,8 @@ class NDT extends NettestBase {
       const uploadMbit = toMbit(summary.upload)
       const downloadMbit = toMbit(summary.download)
       const ping = Math.round(summary.ping*10)/10
-      const packetLoss = Math.round(summary.packet_loss * 100 * 100)/100
-      const outOfOrder = Math.round(summary.out_of_order * 100 * 100)/100
+      const packetLoss = Math.round(summary.packetLoss * 100 * 100)/100
+      const outOfOrder = Math.round(summary.outOfOrder * 100 * 100)/100
```

Comment:
I should probably make a utility function for this.

Reply:
Yes, this should like more useful before than later